hijack/contrib/admin/__init__.py

```diff
@@ -1,6 +1,3 @@
-import django
-
-if django.VERSION < (3, 2):
-    default_app_config = "hijack.contrib.admin.apps.HijackAdminConfig"
-
 from .admin import HijackUserAdminMixin  # noqa
+
+__all__ = ["HijackUserAdminMixin"]
```

Comment:
Let's make this export "explicit" to avoid linters like mypy to raise errors like that:

`.../users/admin.py:18: error: Module "hijack.contrib.admin" does not explicitly export attribute "HijackUserAdminMixin"  [attr-defined]`